docs/docs/development/hardware-integration/index.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Hardware Integration
+sidebar_label: Hardware Integration
+---
+
+import DocCardList from "@theme/DocCardList";
+
+ZMK, being built on [Zephyr™](https://zephyrproject.org/), is very modular and flexible. This allows users to combine features, flags, and properties to "construct" features in ZMK, without the need to modify the underlying source code.
```

Comment:
```suggestion
ZMK, being built on [Zephyr™](https://zephyrproject.org/), is very modular and flexible. This allows users to combine features, flags and properties to "construct" features in ZMK, without the need to modify the underlying source code.
```

Don't want to open the Oxford comma Pandora's box heh, but it doesn't seem necessary to me